tests/test_data.py

```diff
@@ -113,7 +152,7 @@ def test_v3_sensor_parsing():
         back_module_data = {
             "sensors": [
                 {"type": "co2", "value": 35},
-                {"type": "reedVelocity", "value": 20.1},
+                {"type": "reed_velocity", "value": 20.1},
```

Comment:
Also, to test the reed distance field you're adding, can you add some data to the test case?
```json
{"type": "reedDistance", "value": 1040.3},
```